records_mover/records/job/config.py

```diff
@@ -72,24 +77,43 @@ def fill_in_existing_table_handling(self, kwargs: Dict[str, Any]) -> None:
     def fill_in_db_engine(self, kwargs: Dict[str, Any]) -> None:
         kwargs['db_engine'] = self.session.get_db_engine(kwargs['db_name'])
 
-    def fill_in_records_format(self, kwargs: Dict[str, Any]) -> None:
-        if kwargs['variant'] is not None:
```

Comment:
The line above was not necessary as far as I can tell.  When the `variant` command line argument isn't supplied, it doesn't appear in the map, and if it doesn't appear in the map, this function wasn't being called.